src/Generators/AbstractTagGenerator.php

```diff
@@ -215,7 +215,7 @@ protected function generateOwnerTags()
         if ($reflection->isAbstract()) {
             return;
         }
-        if (Injector::inst()->get($this->className) instanceof Extension) {
+        if (is_a($this->className, Extension::class, true)) {
```

Comment:
Why `is_a`? If, for some reason, allowing strings in `is_a` is disabled, this will fail. The reflection class is right above, so `$reflection->isSubclassOf()` makes more sense to use, and is more robust

Reply:
Fair, have made this change now